etc/packaging/appimages/orb-slam3-module-aarch64.yml

```diff
@@ -0,0 +1,77 @@
+# appimage-builder recipe see https://appimage-builder.readthedocs.io for details
```

Comment:
Copy & pasted from https://github.com/viamrobotics/viam-orb-slam3/blob/main/etc/packaging/appimages/orb_grpc_server-aarch64.yml & slightly modified:
```bash
➜  viam-orb-slam3 git:(RSDK-2023-update-orbslam3-appimage) ✗ diff etc/packaging/appimages/orb_grpc_server-aarch64.yml etc/packaging/appimages/orb-slam3-module-aarch64.yml
16a17
>  - cp ../../../bin/orb-slam3-module $TARGET_APPDIR/usr/bin/
21,22c22,23
<     id: com.viam.orb_grpc_server
<     name: orb_grpc_server
---
>     id: com.viam.orb-slam3-module
>     name: orb-slam3-module
44,45d44
<     - libgrpc++1:arm64
<     - libprotobuf31:arm64
49,51d47
<     - libopencv-features2d4.5:arm64
<     - libopencv-imgproc4.5:arm64
<     - libopencv-core4.5:arm64
58,59d53
<     - libglx0:arm64
<     - libgl1:arm64
78,84d71
<     - usr/lib/aarch64-linux-gnu/libzvbi.so.0
<     - usr/lib/aarch64-linux-gnu/libzmq.so.5
<     - usr/lib/aarch64-linux-gnu/libasound.so.2
<     - usr/lib/aarch64-linux-gnu/libvpx.so.6.3
<     - usr/lib/aarch64-linux-gnu/libunistring.so.2
<     - usr/lib/aarch64-linux-gnu/libubsan.so.1
<
87c74
<         AIX_TARGET: usr/bin/orb_grpc_server
---
>         AIX_TARGET: usr/bin/orb-slam3-module
90c77
<   update-information: zsync|http://packages.viam.com/apps/slam-servers/orb_grpc_server-${BUILD_CHANNEL}-aarch64.AppImage.zsync
---
>   update-information: zsync|http://packages.viam.com/apps/slam-servers/orb-slam3-module-${BUILD_CHANNEL}-aarch64.AppImage.zsync

```

Reply:
@zaporter-work found that the following libs were not needed on arm64:
```
libgrpc++1:arm64
libprotobuf31:arm64
libopencv-features2d4.5:arm64
libopencv-imgproc4.5:arm64
libopencv-core4.5:arm64
libglx0:arm64
libgl1:arm64
usr/lib/aarch64-linux-gnu/libzvbi.so.0
usr/lib/aarch64-linux-gnu/libzmq.so.5
usr/lib/aarch64-linux-gnu/libasound.so.2
usr/lib/aarch64-linux-gnu/libvpx.so.6.3
usr/lib/aarch64-linux-gnu/libunistring.so.2
usr/lib/aarch64-linux-gnu/libubsan.so.1
```
and as a result, removed them from the app image 

@JohnN193 do you have any reservations about removing these libs? If so I can add them back.

Reply:
oh nice, I knew there were packages that the appimage def did not need to have included but I was not sure how to filter that out

as long as the appimages can be run in a clean environment then it should be fine with me